Source/WebKit/Shared/ApplePay/mac/WebPaymentCoordinatorProxyMac.mm

```diff
@@ -109,6 +110,7 @@
 
             ASSERT(!paymentCoordinatorProxy->m_sheetWindow);
             paymentCoordinatorProxy->m_sheetWindow = [NSWindow windowWithContentViewController:viewController];
+            paymentCoordinatorProxy->m_sheetWindow.get().styleMask = NSWindowStyleMaskAlertWindow | NSWindowStyleMaskTitled;
```

Comment:
> to be more cohesive with the new platform UI paradigm

If this change is only intended for macOS 26, you'll need to introduce a conditional in `PlatformUse.h` and guard this logic, so it doesn't apply on older macOS.